front/src/common/Map/utils.ts

```diff
@@ -47,3 +47,14 @@ export const onResultSearchClick = ({
     })
   );
 };
+
+export const createMapSearchQuery = (
```

Comment:
Super smart this function ! Maybe it would be useful to add a comment and to rename the parameters like this:
```ts
createMapSearchQuery = (
  searchState: string,
  codeKeyword: string,
  nameKeyword: string,
  isSearchByTrigram?: boolean
) => {}
```
or 
```ts
createMapSearchQuery = (
  searchState: string,
  keywords: { codeKeyword: string, nameKeyword: string },
  isSearchByTrigram?: boolean
) => {}
```